ports/raspberrypi/boards/proveskit_rp2040_v4/mpconfigboard.h

```diff
@@ -0,0 +1,27 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2024 Adafruit Industries LLC
+//
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#define MICROPY_HW_BOARD_NAME "PROVES Kit v4"
+#define MICROPY_HW_MCU_NAME "rp2040"
+
+#define MICROPY_HW_NEOPIXEL (&pin_GPIO24)
+
+#define CIRCUITPY_INTERNAL_NVM_SIZE (4 * 1024)
+#define BOARD_FLASH_SIZE (FLASH_SIZE - 0x4000 - CIRCUITPY_INTERNAL_NVM_SIZE)
+
```

Comment:
No need for these either. They are the default.

```suggestion
```

Reply:
Thank you!